File: python_projects/Beginners/magic8ball.py
# import random

# responses = ["Yes", "No", "Maybe", "Ask again", "Definitely", "I don't think so"]

# question = input("Ask your question: ")
# answer = random.choice(responses)
# print("Magic 8 Ball says:", answer)

import random

responses = [
    "Yes, definitely.",
    "No way!",
    "Ask again later.",
    "It is certain.",
    "I don't think so.",
    "Absolutely!",
    "Not sure, maybe."
]

print("Welcome to the Magic 8 Ball!")
while True:
    question = input("\nAsk a yes/no question (or type 'exit' to quit): ")
    if question.lower() == "exit":
        print("Goodbye!")
        break
    else:
        print("Magic 8 Ball says:", random.choice(responses))

File: java_projects/Beginners/Magic8Ball.java
import java.util.Random;
import java.util.Scanner;

public class Magic8Ball {
    public static void main(String[] args) {
        String[] responses = {
            "Yes", "No", "Maybe", "Ask again", "Definitely", "I don't think so"
        };

        Scanner sc = new Scanner(System.in);
        Random rand = new Random();

        while (true) {
            System.out.print("Ask a question (or type 'exit'): ");
            String question = sc.nextLine();

            if (question.equalsIgnoreCase("exit")) {
                break;
            }

            int index = rand.nextInt(responses.length);
            System.out.println("Magic 8 Ball says: " + responses[index]);
        }

        sc.close();
    }
}
